website/app.py

```
@@ -0,0 +1,171 @@
+@app.route("/docs")
+def docs_redirect():
+    return redirect("/docs/")
+
+
+@app.route("/docs/")
+def docs_index():
+    return send_from_directory("docs", "index.html")
+
+
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=8080)
```

Ruff (0.14.10)

Possible binding to all interfaces

(S104)